test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/InProcTestFhirServer.cs

```diff
@@ -43,6 +43,9 @@ public InProcTestFhirServer(DataStore dataStore, Type startupType)
             configuration["TestAuthEnvironment:FilePath"] = "testauthenvironment.json";
             configuration["FhirServer:Security:Authentication:Authority"] = "https://inprochost";
 
+            // For local development we will use the Azure Storage Emulator for export.
+            configuration["FhirServer:Operations:Export:StorageAccountConnection"] = "UseDevelopmentStorage=true";
```

Comment:
If we can't connect to the DevelopmentStorage emulator locally is there anyway we can include the suggestion for a developer to start it?

Reply:
I did think about that, but wasn't able to find a quick and easy way to do it. The storage emulator is used only for this test. I can maybe call it out in the export documentation.